### ISSUE TITLE
Virtual Paths

### ISSUE THREAD

**Issue by darrencauthon** [lozanotek/mvcturbine#12]
We need to take another look at Embedded Views, it's a little funky when the site is running under a virtual path.

**Comment by jglozano** [comment]
Yes - I wasn't very happy with the implementation for embedded views in the beginning thus my original hesitation to add them.  What's the issue we're experiencing?

**Comment by darrencauthon** [comment]
I _think_ the issue is the resolution of the path.  It doesn't use the relative path when doing the check, so if you're operating under a virtual path your views have to be under that path, too.

For example, if my virtual path is /MANAGE then I have to have a directory in my MVC project called "MANAGE".  If I put my views in that directory, then it works.

I think there are one or two other small things I saw while looking in.  I'm going to take a deeper look at this soon, since I need it in a project I'm working on.  I'm just putting the issue up to let you or anybody else know that we see it and we're looking at it.

**Comment by darrencauthon** [comment]
Has this issue been resolved with @tyronegroves recent pull?
